src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -21,12 +21,11 @@ public Node(Node<T> prev, T item, Node<T> next) {
 
     @Override
     public void add(T value) {
-        Node<T> newNode = new Node<>(tail, value, null);
         if (head == null) {
-            head = tail = newNode;
+            head = tail = new Node<>(null, value, null);
         } else {
-            tail.next = newNode;
-            tail = newNode;
```

Comment:
no, it was ok, but this line can be moved outside if, `cause it's mandatory for this algorithm and will be done in both cases